spec/models/program_spec.rb

```diff
@@ -256,7 +256,7 @@
     it 'returns an array with the types for which a cfp doesn\'t exist' do
       expect(program.remaining_cfp_types).to eq(Cfp::TYPES)
       create(:cfp, cfp_type: 'events', program: program, end_date: Date.current + 1)
-      expect(program.remaining_cfp_types).to eq([])
+      expect(program.remaining_cfp_types).to eq(["booths"])
```

Comment:
You need to amend the test itself, so that you are actually testing what it says it is testing.
Just adding 'booths' here, might make the test pass, but it is not actually testing everything we want to test. 

What are the test cases for the method `remaining_cfp_types`?